Sources/Fuzzilli/Protobuf/sync.pb.swift

```diff
@@ -71,9 +71,9 @@ public struct Fuzzilli_Protobuf_FuzzerState {
   // `Message` and `Message+*Additions` files in the SwiftProtobuf library for
   // methods supported on all messages.
 
-  public var corpus: Data = Data()
+  public var corpus: Data = SwiftProtobuf.Internal.emptyData
```

Comment:
Where does this change come from? Is this from rerunning the swift protobuf generator on the same sync.proto file?

Reply:
Ah my bad, I've just realised the `--experimental_allow_proto3_optional` flag was dropped from the protoc args. Let me re-run protoc without this flag.

Reply:
Ah cool, thanks!

Reply:
So, I've re-run the compiler without the flag and it replaces `Data()` with `SwiftProtobuf.Internal.emptyData`. I'm not sure why this is still `Data()` on the master branch.

Reply:
Ok, seems to be the same for me, so let's just take it :)